README.md

```diff
@@ -97,6 +99,7 @@ Add the following to your `Cargo.toml` file:
 ```toml
 [dependencies]
 serenity = "0.11"
+tokio = { version = "1.21.2", features = ["full"] }
```

Comment:
It's better to enable just the features you need, so `features = ["macros"]` (`#[tokio::main]` is a macro)